lib/rules/require-comments-to-include-access.js

```diff
@@ -5,7 +5,7 @@ function isDocComment(comment) {
 }
 
 function includesAccessDeclaration(comment) {
-  return comment.value.match(/\n\s*(@private|@public)\s/);
+  return comment.value.match(/\n(\s*\*\s|\s*)(@private|@public)\s/);
```

Comment:
Slight tweak here: can you allow multiple (or no) whitespace after the `*`?

I'd like to make sure we match:

```javascript
/**
  *    Some thing smart here as docs...
  *    @public
*/
```

And:

```javascript
/**
  *Some thing smart here as docs...
  *@public
*/
```